textdb/textdb-sandbox/JinggangHelloWorld.java

```diff
@@ -1,6 +0,0 @@
-#include<iostream>
-public class JinggangHelloWorld{
-	public static void main(String[] args){
-		cout << "Hello World!" << endl;
```

Comment:
@diaojinggang  Please remove the .DS_Store file as well.  Thanks for pointing it out @chenlica